services/data/db_utils.py

```diff
@@ -29,25 +32,13 @@ def get_db_ts_epoch_str():
 
 
 def translate_run_key(v: str):
-    key = "run_id"
-    value = "'{0}'".format(v)
```

Comment:
i need to double check but I believe the single quotation marks are necessary when querying the db based off run_id

Reply:
I'll check this as well

Reply:
Single quotation marks are not necessary due to the way we construct queries from now on at `postgres_async_db.py` https://github.com/Netflix/metaflow-service/pull/144/files/01e89fa7966259c89be9e13aca82e4cfbb93da64#diff-6bca962443914b058b03743877f9eab3f0f30804c06c086146d53d2aee6aa88aR220

This allows Psycopg handle different data types and makes it safer as well, more information here: https://www.psycopg.org/docs/usage.html#the-problem-with-the-query-parameters